src/salah/times.rs

```diff
@@ -312,7 +319,7 @@ impl PrayerTimes {
     fn current_time(&self, time: DateTime) -> Prayer {
         // dummy value. it will replaced below
         // just to avoid using `Option` or `Err`
-        let mut current_prayer = Prayer::Dohr;
+        let mut current_prayer = Prayer::Ishaa;
```

Comment:
Would you like to make sure this change doesn't cause regressions?

Or, make sure that the failed test is valid. I use `https://www.jadwalsholat.org/`
See https://github.com/azzamsa/islam/blob/7f19f2274c8da578bb92d9dd090eec5f9a3d301c/src/salah/times.rs#L429, https://github.com/azzamsa/islam/blob/7f19f2274c8da578bb92d9dd090eec5f9a3d301c/src/salah/times.rs#L377

Reply:
Fixed test current_prayer_is_dohr()
by replacing expected_time method with expected_time_with_date.
Other prayer test is using the later one, I wonder why only the Dohr using expected_time.

let current_prayer_time = expected_time_with_date(date, 11, 52, 0)?;